docs/source/release/v6.6.0/framework.rst

```diff
@@ -7,53 +7,64 @@ Framework Changes
 
 New Features
 ------------
-.. amalgamate:: Framework/New_features
+- the approach for inverting large tridiagonal matrices has been improved so that the analytical method can still be used even for large matrices, resulting in a performance improvement. This type of matrix inversion is used when calculating the interpolation errors when performing spline interpolation
+- HDF5 has been upgraded to v1.12 along with h5py to version 3.
 
 Algorithms
 ----------
 
 New features
 ############
-.. amalgamate:: Framework/Algorithms/New_features
+- To allow files saved with :ref:`SaveAscii2 <algm-SaveAscii-v2>` to be reloaded into Mantid and preserve rebinning behaviour, a ``Distribution=true`` flag is now written in the file header if the workspace is a Distribution. In this case, a column header is always written, even if the Save property ``ColumnHeader`` is set to false.
+- To load an Ascii file, that was saved without a Distribution header flag, and set it as a Distribution, execute :ref:`LoadAscii2 <algm-LoadAscii-v2>` with property ``ForceDistributionTrue=true``
+- Add error calculation to :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` following similar approach to :ref:`algm-MonteCarloAbsorption`
+- The `LoadSANS1MLZ  <https://docs.mantidproject.org/nightly/algorithms/LoadSANS1MLZ.html>`_ algorithm now can be used to load raw files with extension ``.001`` generated by the SANS-1 instrument at MLZ, as well as to load reduced files with extension ``.002``.
+- Added support for removal of instrument information from workspace - see :ref:`RemoveInstrumentGeometry <algm-RemoveInstrumentGeometry>`. This will allow saving smaller nexus files.
+- remove the MuscatData, MuscatFunc and MuscatSofQW algorithms. The functionality provided by these algorithms is now provided by the :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` algorithm
+- Add support for a beam profile (specified in :ref:`SetBeam <algm-SetBeam>`) to :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>`
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- The algorithm GUI for UpdateInstrumentFromFile now accepts .DAT files for the Filename property.
+- Reworked loadPropertyCommon to fix an issue with loading MDE worksapces taking long time to load
```

Comment:
There's a typo here - "worksapces" should be "workspaces". While we're here, we could consider re-writing this as "workspaces taking a long time to load"